ui/Hotkeys.svelte

```diff
@@ -48,6 +48,13 @@
 
     if (!shortcut) return;
 
+    // Close any opened modal, before openening the next one.
+    modal.hide();
```

Comment:
This leads to a slight flicker, it might be more effective to move this logic into the toggle and only hide the modal if the new destination isn't one.

Reply:
For me the flicker only happens, when I have a modal opened and the switch to a fullscreen (settings or design system guide). I don't see a flicker when switching between modals.
For the case modal-fullscreen case I guess we have to decide on the order, either close modal than navigate (that is how it is now) or first navigate and then close the modal.

Reply:
Sorry the case where we navigate to a fullscreen page is the case I meant. The second option you layed out seems reasonable and is what I tried to say with checking if it is a modal that we navigate to before hiding.